src/fetching/views/getViewColumns.ts

```diff
@@ -0,0 +1,10 @@
+import { defaultFetch } from '..';
+import { ZetkinViewColumn } from '../../types/zetkin';
+
+export default function getViewColumns(orgId : string, viewId : string, fetch = defaultFetch) {
+    return async () : Promise<ZetkinViewColumn[]> => {
+        const oRes = await fetch(`/orgs/${orgId}/people/views/${viewId}/columns`);
```

Comment:
Esp for these client side requests, we should throw an error if the response is not a 200 status. I made a new helper util in the last PR, `APIError`, which I made to standardise the error messages in this situation. I haven't replaced other error messages with it yet, but it's possibly good to use going forward.